sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/Offer.java

```diff
@@ -26,6 +27,18 @@ public Offer(int offerThroughput) {
         this.setContent(content);
     }
 
+    Offer(OfferAutoscaleSettings offerAutoscaleSettings) {
+        super();
+        this.setOfferVersion(Constants.Properties.OFFER_VERSION_V2);
```

Comment:
autopilot is a v2 offer?

Reply:
Yes it is according to the offer payload I saw in Autopilot doc.